test/config/autotls/certmanager/caissuer/secret.yaml

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Does yaml require /bin/bash :thinking:

Reply:
Thank you! No, it was a mistake when I copied the copyright header from script :smile: